### ISSUE TITLE
SyntaxError on Ubuntu

### ISSUE THREAD

**Issue by bitcoin4cashqc** [kitao/pyxel#124]
**Describe the bug**
Trying to install on my ubuntu machine, did compiled GLFW manually, python3 as well. 

Got this:
`pyxeleditor -hTraceback (most recent call last):
  File "/usr/local/bin/pyxeleditor", line 7, in <module>
    from pyxel.editor import run
  File "/usr/local/lib/python3.5/dist-packages/pyxel/__init__.py", line 12
    DEFAULT_BORDER_COLOR: int = constants.DEFAULT_BORDER_COLOR
                        ^
SyntaxError: invalid syntax
`

I also tried in pyenv and got the GLFW wrong version issue

Anyone able to install on Ubuntu or even better, pyenv?

**Desktop (please complete the following information):**
 - Ubuntu
 - python3, python3.7.0

**Comment by bitcoin4cashqc** [comment]
With pyenv : 
`RuntimeError: glfw version is lower than 3.2.1`

and in the pip:
Collecting glfw (from pyxel)
  Downloading https://files.pythonhosted.org/packages/5d/65/c6275744a01425195f1f446e022e5dfa6497aa68479a3952e434e04b2fa0/glfw-1.7.1.tar.gz

**Comment by bitcoin4cashqc** [comment]
Anyone have an idea how to actually run pyxel under a pyenv

**Comment by bitcoin4cashqc** [comment]
Ok so

I was able to open pyxeleditor from pyenv without GLFW issues, but I needed to start it with sudo, which make the example and the module itself break with my system python.

So I installed python3.7.0 side with my system python3, recompiled GLFW with the -DBUILD_SHARED_LIBS=ON  (not sure if needed),  installed pyxel, and the editor work, and the module itself look to work!

Finally ! This look so freaking nice.
